src/i18n/index.ts

```diff
@@ -26,7 +26,13 @@ const resources = {
 };
 const namespace = 'translation';
 
-export const languages = [
+export interface LanguageDefinition {
+  code: string;
+  name: string;
+  flag: string;
+}
+
+export const languages: LanguageDefinition[] = [
```

Comment:
Fancy sort function...

I would have also accepted just manually ordering this array ;)

Reply:
This way nobody needs to remember adding new languages in a specific order. Those things tend to be forgotten over time :)